labs/spring_2016_update.md

```diff
@@ -0,0 +1,29 @@
+Subject: You Are Now Hack Club's Ambassadors
+
+Hi everyone!
+
+We've learned and failed a tremendous amount this past semester. All of you
+have been incredibly supportive and both of us feel so privileged to have the
+opportunity to work with each and every one of you.
+
+In the next two months, Hack Club is focusing on growth. Our goal is to start
+50 clubs and increase our weekly attendance by 750 students by the end of
+February. As a result of this, we're focusing on rewriting our early workshops
+and standardizing the beginning of clubs ([click here][club_guide] for an
+example of what we're working on). This means that we won't be building
+curriculum for clubs like yours at the pace we did last semester and we may
+not have new workshops in time for all of your meetings while we're still
+early into this semester.
+
+[club_guide](https://github.com/hackclub/hackclub/blob/master/clubs/README.md)
```

Comment:
Reference link definitions should go at the bottom of the section (and in this case, the file). Can you move this to the very bottom?